doc/api.yml

```diff
@@ -877,6 +1145,17 @@ components:
             $ref: "#/components/schemas/Category"
         reactions:
           $ref: "#/components/schemas/ReactionStat"
+
+    PostPicture:
+      properties:
+        id:
+          type: string
+        post:
+          $ref: "#/components/schemas/Post"
+        document_reference:
```

Comment:
Is this used in the frontend?

Are these informations really necessary?

I think the only thing that is going to be used is the picture url
> Unless there is a specific use-case you want to address here

Reply:
document_reference serves as a placeholder that the front puts in the content (document) when they create/update a Post, so we can replace them later when someone requests a Post. 
I'll change the name to "placeholder"

Reply:
Just to be certain, they're replaced (mapped) when someone requests a Post? we don't need to replace them manually in the frontend ?